.changesets/fix_bryn_json_schema.md

```diff
@@ -0,0 +1,7 @@
+### Make json schema small ([PR #5061](https://github.com/apollographql/router/pull/5061))
+
+The json schema for the router.yaml is reduced in size from approx 100k lines to just over 7k.
+
+This reduces the startup time of the Router and a smaller schema is more friendly for code editors.
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/5061
```

Comment:
@BrynCooke Let's call this what it is.  We introduced a bug that caused a baseline memory footprint increase in Router 1.46.0. (And reference the issue)

Reply:
text was changed